package.json

```diff
@@ -26,6 +26,7 @@
     "lodash.isstring": "^4.0.1"
   },
   "devDependencies": {
+    "@justinbeckwith/typedoc": "^0.10.1",
```

Comment:
You published typedoc under your name? How is it different from the `typedoc` package?

Reply:
Yup, covered this above. TypeDoc has a ton of un-merged PRs, and it doesn't work today with TypeScript 2.6.  I had to go create my own fork, update a bunch of dependencies, add TypeScript 2.6 support, and publish to npm. That's why it's @justinbeckwith/TypeDoc :)  When they get their issues sorted out, we should try to move back.

Reply:
Got it. BTW https://www.npmjs.com/package/@justinbeckwith/typedoc points to https://github.com/TypeStrong/TypeDoc as its repo, not yours. Is it intentional?